Sources/App/Views/PublicPage.swift

```diff
@@ -191,6 +192,12 @@ class PublicPage {
             stagingBanner()
         )
     }
+
+    /// Any page level HTML comments for hidden metadata.
+    /// - Returns: An element, or `group` of elements.
+    func bodyComments() -> Node<HTML.BodyContext> {
+        .text("")
```

Comment:
could use `.empty` here (doesn't seem to change snapshots)

Reply:
Oh, empty is so much better! We use `.text("")` in a few places so I'll standardise in a separate PR.